### PR TITLE
Much faster Transit Tubes, Still Pause a Pause a Good Amount of Time for Boarding

### PR DESCRIPTION
**Pull request by DatBoiTim**
## About The Pull Request
Makes Transit Tubes actually decent for transportation

## Why It's Good For The Game
They kinda suck. Make em faster!

## Testing Photographs and Procedure

Go to a map that has transit tubes. Get in and go for a spin!

## Changelog
:cl: DatBoiTim
tweak: Speeds up Transit Pods
/:cl:

### CODE REVIEW

**Comment by DatBoiTim** [comment]
This may or may not be meant to make Tube Station bearable.

**Comment by DatBoiTim** [comment]
For reference they are faster than base run speed

File: code/game/objects/structures/transit_tubes/transit_tube.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	pass_flags_self = PASSGLASS
 	var/tube_construction = /obj/structure/c_transit_tube
 	var/list/tube_dirs //list of directions this tube section can connect to.
-	var/exit_delay = 1
+	var/exit_delay = 0.25
```
Comment (LemonInTheDark) [inline code/game/objects/structures/transit_tubes/transit_tube.dm:14]: I don't think setting this to anything below 0.5 really does anything, can't sleep for half a tick after all

Comment (DatBoiTim) [reply code/game/objects/structures/transit_tubes/transit_tube.dm:14]: afd12d4

File: code/game/objects/structures/transit_tubes/transit_tube.dm [hunk 2]
```diff
@@ -11,7 +11,7 @@
 	pass_flags_self = PASSGLASS
 	var/tube_construction = /obj/structure/c_transit_tube
 	var/list/tube_dirs //list of directions this tube section can connect to.
-	var/exit_delay = 1
+	var/exit_delay = 0
```
Comment (Raven-Industries) [inline code/game/objects/structures/transit_tubes/transit_tube.dm]: Setting it to zero is essentially equivalent to teleportation, at least when I tried it.

**Review by PowerfulBacon — COMMENTED** [review]
The reason pods are so slow is because they sit at stations for like 5 seconds before moving. Even if you double the speed of the pods, they will still spend 80% of their time sitting at a station doing nothing.

**Comment by DatBoiTim** [comment]
Didn't even notice this got closed.

> Even if you double the speed of the pods, they will still spend 80% of their time sitting at a station doing nothing.

While this won't change smaller tube system speeds, it certainly increases speed for larger ones, allowing mapmakers to have interesting options for creating maps(Tubestation for example). The idea is to move most of the wait time for a pod to be the load/unload and not the actual moving of the pod.

**Pull request closed without merging**